go.mod

```diff
@@ -82,6 +82,7 @@ require (
 	k8s.io/apiextensions-apiserver v0.27.9
 	k8s.io/apimachinery v0.27.11
 	k8s.io/client-go v0.27.11
+	k8s.io/klog v1.0.0
```

Comment:
Odd ... we shouldn't be depending on klog v1

Reply:
good catch. I have some local file not checked in to this PR. That may be why this lib is introduced.